lib/plurimath/math/function/mod.rb

```diff
@@ -7,8 +7,8 @@ module Math
     module Function
       class Mod < BinaryFunction
         def to_asciimath
-          first_value = parameter_one.to_asciimath if parameter_one
-          second_value = parameter_two.to_asciimath if parameter_two
+          first_value = wrapped(parameter_one) if parameter_one
+          second_value = wrapped(parameter_two) if parameter_two
```

Comment:
The first line in the `wrapped` function is `return "" unless field`. Do we need an `if` condition here?